tests/server/master_server/lock_manager_unit_test.cc

```diff
@@ -4,33 +4,38 @@
 #include "gtest/gtest.h"
 #include "src/server/master_server/lock_manager.h"
 
+using namespace gfs::server;
+
 class LockManagerUnitTest : public ::testing::Test {
  protected:
   void SetUp() override {
-    lockManager_ = gfs::server::LockManager::GetInstance();
+    lockManager_ = LockManager::GetInstance();
   }
 
-  gfs::server::LockManager* lockManager_;
+  LockManager* lockManager_;
 };
 
+// Add lock step by step and make sure the exist
```

Comment:
How about tests for trying to add locks that already exists, or fetch locks that don't exist?

Reply:
They are covered in this change. See later test case